apis/nucleus/src/components/Supernova.jsx

```diff
@@ -19,7 +19,7 @@ const VizElement = {
   className: 'njs-viz',
 };
 
-function Supernova({ sn, snOptions: options, snPlugins: plugins, layout, appLayout, halo }) {
+function Supernova({ sn, snOptions: options, snPlugins: plugins, layout, appLayout, halo, cellId }) {
```

Comment:
@Caele Could this function be used without going through cell.jsx?

Reply:
yes, but only inside current client. It is only expose through here: https://github.com/qlik-oss/nebula.js/blob/main/apis/stardust/src/index.js#L48, the `__DO_NOT_USE__`.